projects/components/src/datagrid/renderers/linked-text-renderer.component.ts

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
This can needs some work until it's actually useful. It's not worth maintaining it (adding unit tests). Let's remove this, and add unit tests for the `BoldTextRenderer` which can be used as an example until we have more useful renderers.